compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -7847,11 +7982,13 @@ TR::Node *dsubSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    // Pretend to be an int before doing the comparison
    //
    // According to the java spec -0.0 - (-0.0) == 0.0
-   // BINARY_IDENTITY_OP(LongInt, DOUBLE_NEG_ZERO)
    // In IEEE FP Arithmetic, d - -0.0 is d
    // Pretend to be an int before doing the comparison
    //
-   BINARY_IDENTITY_OP(LongInt, DOUBLE_POS_ZERO)
+   auto binOpSimplifier = getDoubleBinaryOpSimplifier(s);
```

Comment:
The original code calls macro `BINARY_IDENTITY_OP(LongInt, DOUBLE_POS_ZERO)` which calls `secondChild->getLongIntType()`. Should the corresponding new API be `getLongBinaryOpSimplifier`?